### ISSUE TITLE
BertForTokenClassification for NER, mask labels 

### ISSUE THREAD

**Issue by alexyalunin** [huggingface/pytorch-pretrained-BERT#422]
I'm trying to do Named Entity Recognition with BertForTokenClassification.
Say I have 10 words with 10 labels, after WordPiece tokenization I get 15 tokens and I assign them labels, "X" for pieces of words like (##ing).
In the original paper https://arxiv.org/pdf/1810.04805.pdf in 4.3 section is said that we don't make predictions for such pieces or we can simply do not care about them and not take derivatives. 
Does this functionality currently exist in BertForTokenClassification or BertModel?

**Comment by bheinzerling** [comment]
Sequence tagging is explained here: https://github.com/huggingface/pytorch-pretrained-BERT/issues/64#issuecomment-443703063

**Comment by thomwolf** [comment]
Yes, this is the relevant issue on this topic. I'll close this issue in favor of #64.
